Makefile

```diff
@@ -687,6 +687,29 @@ gh-install-pin-github-action:
 gh-pin-actions: gh-install-pin-github-action
 	@pin-github-action ./.github/workflows/release.yaml
 
+#############
+# PERF TEST #
+#############
+
+.PHONY: test-perf
+test-perf: $(PACKAGE_SHIM)
+	GO111MODULE=off GOPATH=$(GOPATH_SHIM) go get k8s.io/perf-tests || true
+	cd $(GOPATH_SHIM)/src/k8s.io/perf-tests && \
+	GOPATH=$(GOPATH_SHIM) ./run-e2e.sh cluster-loader2 \
+		--testconfig=./testing/load/config.yaml \
+		--provider=kind \
+		--kubeconfig=${HOME}/.kube/config \
+		--nodes=3 \
```

Comment:
Thanks @eddycharly !

A 3 nodes cluster for performance tests doesn't seem to be enough, is this configurable?

Reply:
What do you mean by _doesn't seem to be enough_ ?
The tool adapts to the number of nodes.

Reply:
I meant a 3 nodes cluster can only take up to 300 pods, this is not enough for performance testing.

Reply:
Ah ok, this is not configurable yet.

Reply:
Made it configurable.